addons/im_livechat/static/tests/sidebar_patch.test.js

```diff
@@ -384,6 +388,7 @@ test("unknown livechat can be displayed and interacted with", async () => {
     await insertText(".o-mail-Composer-input", "Hello", { replace: true });
     await click(".o-mail-Composer-send:enabled");
     await contains(".o-mail-Message", { text: "Hello" });
+    await waitNotifications([env, "discuss.channel/new_message"]);
```

Comment:
Ok for now. I think this function could be removed if we make some UI distinction with temporary / actual message. For example, the date could be displayed only for genuine message. Doing so would allow to `await` the right time, without being too technical like `waitNotifications` that has no UI feedback.

Reply:
Didn't we have a class for temporary message that gets removed when it becomes confirmed by server?
Or that was only in a previous version?